tests/MiniProfiler.Tests/App.config

```diff
@@ -34,4 +34,7 @@
       <add name="SQLite Data Provider" invariant="System.Data.SQLite" description=".NET Framework Data Provider for SQLite" type="System.Data.SQLite.SQLiteFactory, System.Data.SQLite" />
     </DbProviderFactories>
   </system.data>
+  <appSettings>
+    <add key="xunit.parallelizeTestCollections" value="false"/>
```

Comment:
this is true, however i think this might be pretty tough to do with the mini profiler library tests. The library relies on a static variable to keep the profiler provider and some of the tests replace profiler providers, if a provider is replaced in a middle of test's execution it might result in some fairly weird failures. I think pretty much all tests would need to be in the same category and in future it would be a pain to maintain the categorization, since the requirements are not quite obvious. 

And even with the parallelization disabled the unit tests execute pretty fast.